source/set-up/shared-channels.rst

```diff
@@ -0,0 +1,193 @@
+
+Alternatively, you can extend a read-only invitation to a secure connection by appending the optional --readonly parameter to this command. Remote members can’t post or reply to messages within shared read-only channels.
+
+.. tip:: 
+
+    You can re-extend an invitation to a secure connection by omitting the optional ``--readonly`` parameter that also enables remote member participation in that channel.
+
+For example:
+
+``/share-channel invite --connectionID "AcmeUS"``
+ 
+This slash command invites AcmeUS to the current channel.
+
+.. tip:: 
+    See `Reviewing Secure Connection Status <#reviewing-secure-connection-status>`_ to find the connectionID for a shared connection.
+
+Uninviting Shared Channel Connection from a Channel
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Within a specific channel, use the following slash command to uninvite a secure connection:
+
+``/share-channel uninvite --connectionID``
+
+This slash command removes a secure connection from the current channel based on its Connection ID. The channel continues to function for local users as expected., and the secure connection may continue to be invited to other shared channels.  
+
+Removing All Secure Connections from a Channel
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Within a specific channel, use the following slash command to remove all secure connections: 
+
+``/share-channel unshare``
+
+This slash command removes all secure connections from the current channel. Secure connections may continue to be invited to other shared channels. 
+
+While unsharing a shared channel stops synchronizing the channel with the other Mattermost server, the channel continues to function for local users as expected. 
+
+.. note:: 
+    A System message notifies System Admins that the channel is no longer shared.
+
+Reviewing Secure Connections in Channels
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Use the following slash command to review the status of all secure connections within the current shared channel:
+
+``/share-channel status``
+
+Status details include:
+- Connection ID
+- Connection URL
+- Description
+- Read only channel (True/False)
+- Invite accepted (Yes/No)
+- Online (Yes/No)
+- Last ping timestamp (UTC)
+
+Frequently Asked Questions
+---------------------------
+
+Why is this feature in Beta?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+This feature is in beta while we recruit customer testing partners. This feature has been tested by Mattermost QA, but as we build the interface for managing shared channels, we want to work with System Admins to build the most optimal experience.
+
+Are special characters supported in secure connection names?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+No. ``--name`` can include periods, hyphens, and/or underscores. You must surround ``--name`` and ``--displayname`` values using quotation marks (“ “) when the values contain spaces.
```

Comment:
There seems to be a bug in the slash command parsing code that doesn't handle spaces for the display name, even when quoted.  This is not specific to shared channels - it's a general slash command bug.